$NetBSD: patch-ApiExtractor_parser_rpp_pp-qt-configuration,v 1.1 2013/11/15 12:14:47 rhialto Exp $

__STDC__ should be defined as "1". <sys/cdefs.h> depends on that.

--- ApiExtractor/parser/rpp/pp-qt-configuration.orig	2013-11-12 00:25:51.000000000 +0000
+++ ApiExtractor/parser/rpp/pp-qt-configuration
@@ -1,6 +1,6 @@
 #define __cplusplus 1
 
-#define __STDC__
+#define __STDC__ 1
 
 // Qt
 #define QOBJECTDEFS_H
